src/Exception/NotFoundException.php

```diff
@@ -12,19 +12,21 @@
 final class NotFoundException extends Exception implements NotFoundExceptionInterface, FriendlyExceptionInterface
 {
     #[Pure]
-    public function __construct(string $class, string $message = null, int $code = 0, Exception $previous = null)
+    public function __construct(string $class, ?string $message = null, int $code = 0, ?Exception $previous = null)
```

Comment:
```suggestion
    public function __construct(string $class, string|null $message = null, int $code = 0, Exception|null $previous = null)
```
?

Reply:
This and other packages used the restriction of types `?type`, not `type|null`